app/scripts/proactive/model/Task.js

```diff
@@ -58,13 +58,21 @@ define(
                   },
                   title: 'Documentation'
                 },
-
+                "Task Result Added to Job Result": {
+                    type: "Checkbox",
+                    fieldAttrs: {
+                        'placeholder': '@attributes->preciousResult',
+                        "data-help": 'Indicates if you want to save the result of this task in the job result.'
+                    }
+                },
                 "Variables": {
                     type: 'List',
                     itemType: 'Object',
                     title: "Task Variables (Name, Value, Inherited)",
                     fieldAttrs: {
                         'placeholder': 'variables->variable',
+                        "data-tab": "Task Variables",
```

Comment:
why not Variables ?

Reply:
Here is the tab name, I feel it might be better to clarify it is just the "Task Variables"?

Reply:
maybe you should ask d.c. before merging it ?